src/SConscript

```diff
@@ -7,7 +7,6 @@ HEADERS = ['daos_api.h', 'daos_types.h', 'daos_errno.h', 'daos_kv.h',
            'daos_task.h', 'daos_fs.h', 'daos_uns.h', 'daos_security.h',
            'daos_prop.h', 'daos_obj_class.h', 'daos_obj.h', 'daos_pool.h',
            'daos_cont.h', 'daos_version.h', 'daos_fs_sys.h', 'daos_s3.h', 'daos_pipeline.h']
-HEADERS_SRV = ['vos.h', 'vos_types.h']
```

Comment:
Maybe I missed it - how are these being installed now?

Reply:
We're not.